inc/Editor/BlockManagement/ConfigRegistry.php

```diff
@@ -23,15 +23,57 @@ class ConfigRegistry {
 	public const RENDER_QUERY_KEY = 'render_query';
 	public const SELECTION_QUERIES_KEY = 'selection_queries';
 	public const DISPLAY_QUERY_KEY = 'display';
+	public const DISPLAY_QUERIES_KEY = 'display_queries';
 	public const LIST_QUERY_KEY = 'list';
 	public const SEARCH_QUERY_KEY = 'search';
+	public const QUERIES_KEY = 'queries';
 
 	public static function init( ?LoggerInterface $logger = null ): void {
 		self::$logger = $logger ?? new Logger();
 		ConfigStore::init( self::$logger );
 	}
 
+	private static function migrate_block_config( array $block_config = [] ): array|WP_Error {
```

Comment:
Seems like we should have actual type validation of block config in a class that extends `ArraySerializable`

Reply:
I did this in 82b30540

Reply:
Thanks for making that change!